hoomd/md/nlist.py

```diff
@@ -85,7 +87,7 @@ class NeighborList(AutotunedObject):
     """
```

Comment:
Before this gets fixed in stone, would you maybe prefer that it be called `base_cut` or `base_r_cut` instead of `r_cut`. In addition to documenting this clearly, this may help clear up an obvious confusion with having the same name in multiple places doing different things.